src/components/pageComponents/ContentCard.js

```diff
@@ -90,8 +94,15 @@ export default function ContentCard(props) {
   }
 
   const handleFavoriteClick = () => {
-    setFavorite(!favorite);
-    // todo: save state to backend
+    if (favorite) {
+      deleteFavoriteItem(data.id, "701b389b848a2b1cfab867093101d8d5ac56addd")
```

Comment:
I think you can get the id of the current user via the authentication class get userId() as far as I understand the code